book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] @Tafloninno great work adding the add_rental method, however, this method is supposed to take three parameters `date, self, person` (self being the book itself) this will ensure we achieve `the 'has-many' association`.